com.unity.multiplayer.mlapi/Runtime/Configuration/MLAPIConstants.cs

```diff
@@ -64,8 +66,8 @@ internal static class MLAPIConstants
             "",
             "",
             "",
-            "",
-            "",
+            "MLAPI_STD_SERVER_RPC",
```

Comment:
Um, ok, I guess we wanted to ensure these are 30 & 31?

Reply:
yeah, we could get rid of this list but it's still around. string index is matching their const int IDs too but I think I can move them up in this list and update their message IDs.